luma/core/bitmap_font.py

```diff
@@ -0,0 +1,632 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017-18 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+import cbor2
+from pathlib import Path
+from PIL import Image, ImageFont, UnidentifiedImageError
+from math import ceil
+from copy import deepcopy
+
+
+class bitmap_font():
+    """
+    An PIL.Imagefont style font
+
+    The structure of this class was modeled after the PIL ImageFont class
+    and is intended to be interchangable for PIL.ImageFont objects.  It has
+    the following additional capabilities.
+
+    * Allows fonts larger than 256 characters to be created
+    * Font can be combined with other fonts
+    * Font characters can be mapped to their correct unicode codepoints
+    * Font can be initialized from a basic sprite table (with conditions)
+
+    .. note:
+        Because this font is implemented completely in Python it will be slower
+        than a native PIL.ImageFont object.
+
+    .. versionadded:: 1.15.1
+    """
+
+    PUA_SPACE = 0xF8000
+
+    def __init__(self):
+        self.mappings = {}
+        self.metrics = []
+
+    def load(self, file):
+        """
+        Load luma.core.bitmap_font
```

Comment:
I wouldn't reference this classname itself in the docstring. Also use something like `fname` and document it as a string.